mapcomposer/app/applications/geocollect/static/gctranslations/it.js

```diff
@@ -0,0 +1,270 @@
+        centerMsgLabel:"Messaggio",
+        localizeFieldLabel:"Localizza",
+        localizeMsgLabel:"Messaggio",
+        zoomLevelLabel:"Levelli Zoom",
+        mapHeightLabel:"Altezza Mappa"
+    },
+    "mxp.widgets.XtypeLabel":{
+        idFieldLabel:"Campo",
+        labFieldLabel:"Label",
+        valueFieldLabel:"Valore"
+    },
+    "mxp.widgets.XtypeDateField.prototype":{
+        idFieldLabel:"Campo",
+        labFieldLabel:"Label",
+    }, "mxp.widgets.XtypeCheckBox.prototype":{
+        idFieldLabel:"Campo",
+        labFieldLabel:"Label"
+    },
+    "mxp.widgets.XActionSend.prototype":{
+        valueFieldLabel:"Testo",
+        labFieldLabel:"Messaggio"
+    },
+    "mxp.widgets.XActionSave.prototype":{
+        valueFieldLabel:"Testo",
+        labFieldLabel:"Messaggio",
+    },
+    //GeoCollect Composer Classes
+     "gxp.plugins.GcSegGrid.prototype": {
+        displayFeatureText: "Mostra In  Mappa",
+        firstPageTip: "Prima Pagina",
+        previousPageTip: "Pagina Precedente",
+        zoomPageExtentTip: "Zoom All'Estensione Della Pagina",
+        nextPageTip: "Prossima Pagina",
+        lastPageTip: "Ultima Pagina",
+        title: "Entità",
+        totalMsg: "Totale: {0} Entità",
+        displayExportCSVText: "Esporta Nel Formato CSV",
+        exportCSVSingleText: "Singola Pagina",
+        exportCSVMultipleText: "Tutte Le Pagine",
+        failedExportCSV: "Impossibile Esportare Nel Formato CSV",
+        invalidParameterValueErrorText: "Valore Parametro Non Valido",
+        zoomToFeature: "Zoom Alla Feature",
+        exportTitleText: "Esporta",
+        pageLabel: "Pagina",
+        pageOfLabel: "di",
+        totalRecordsLabel: "Elementi Totali",
+        btnDetailsText:  "Dettagli",
+        btnDetailsTooltip: "Apri Dettaglii Segnalazione",
+        btnMapText:  "Mappa",
+        btnMapTooltip: "Mostra Mappa",
+        noticeDetailsPanelTitle:"Dettgli Segnalazione",
+        photoBrowserPanelTitle:"Foto Sopralluogo",
+        noticePhotoBrowserPanelTitle:"Foto Segnalazione",
+        surveysPanelTitle:"Sopralluoghi",
+        noticePanelTitle:"Segnalazione",        
+    },
+    "gxp.plugins.GcFeatureEditor.prototype":{
+        createFeatureActionTip: "Crea Nuova Segnalazione",
+        saveOrCancelEdit:'Salva O Elimina Modifiche',
+        editFeatureActionTip: "Seleziona Segnalazione",
+
+    },
+    "gxp.grid.GcHistoryGrid.prototype":{
+        title:'Storico',
+        loadingText: 'Aspetta...Carico Storico',
+        emptyText: "Ricerca",
+    },
+    "Ext.PagingToolbar.prototype":{
+        displayMsg : 'Mostrando {0} - {1} di {2}',
+        emptyMsg : 'Nessun Dato',
+        beforePageText : 'Pagina',
+        afterPageText : 'di {0}',
+        firstText : 'Prima Pagina',
+        prevText : 'Page Precedente',
+        nextText : 'Prossima Pagina',
+        lastText : 'Ultima Pagina',
+        refreshText : 'Aggiorna'
+    },
+    "gxp.plugins.GcSegForm.protorype":{
```

Comment:
"protorype" instead of "prototype" prevents the translations to be applied correctly